.goreleaser.yaml

```diff
@@ -0,0 +1,26 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+builds:
+  - id: aws-auth
+    main: ./
+    binary: aws-auth
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    ldflags:
+      - -s -w -X github.com/keikoproj/aws-auth/cmd/cli.gitCommit={{.Commit}} -X github.com/keikoproj/aws-auth/cmd/cli.buildDate={{.Date}} -X github.com/keikoproj/aws-auth/cmd/cli.pkgVersion={{.Version}}
```

Comment:
Addon manager uses goreleaser too https://github.com/keikoproj/addon-manager/blob/master/.goreleaser.yml

The `ldflags` can be multi line